spawn.yml

```
@@ -9,3 +9,45 @@
 
     - name: create_multinode
       when: not is_aio
+
+  tasks:
+    - name: Load inventory file
+      ansible.builtin.command: ansible-inventory -i kolla-inventory --list
+      register: inventory_content
+
+    - name: Parse the inventory map
+      set_fact:
+        inventory_map: "{{ inventory_content.stdout | from_json }}"
+
+    - name: Get deployment host IP
+      set_fact:
+        deployment_hosts: "{{ inventory_map['deployment']['hosts'] }}"
+
+    - name: Create ara_deployments group
+      ansible.builtin.add_host:
+        hostname: "{{ item }}"
+        ansible_private_key_file: "{{ kolla_ssh_key }}"
+        group: ara_deployments
+      loop: "{{ deployment_hosts }}"
+
+- name: Configure ARA recording on deployment nodes
+  hosts: ara_deployments
+  user: kolla
+  gather_facts: false
+
```

```suggestion
    vars:
      ansible_ssh_common_args: '-o IdentitiesOnly=yes'
      
```
Or just move all this to `prepare` playbook.

---

Moved it to prepare playbook.